packages/pwa-kit-runtime/src/ssr/server/constants.js

```diff
@@ -7,7 +7,6 @@
 export const APPLICATION_OCTET_STREAM = 'application/octet-stream'
 export const BUILD = 'build'
 export const STATIC_ASSETS = 'static_assets'
-export const PROXY_PATH_PREFIX = '/mobify/proxy'
```

Comment:
Technically a breaking change. Is there a way that we could NOT a breaking change?

Reply:
I restored the constants that I removed and annotated them with a `@deprecated`. Let me know if this works